beater/api/asset/sourcemap/handler.go

```diff
@@ -46,8 +47,14 @@ var (
 	validateError = monitoring.NewInt(registry, "validation.errors")
 )
 
+// AddedNotifier is an interface for notifying of sourcemap additions.
+// This is implemented by sourcemap.Store.
+type AddedNotifier interface {
```

Comment:
What's the motivation for exporting this interface? It doesn't cost anything to export it, just curious since the interface isn't being referenced as an argument or struct member anywhere else.

Reply:
It's used in the Handler function below. Am I misunderstanding your question?

Reply:
Going to merge as-is, happy to update if you think there's an improvement to be had.

Reply:
I don't think it's an improvement, just wondering your personal opinion about exporting or not exporting an interface. In this case, `AddedNotifier` is only referenced once in the project, as an argument in the below function in the same package. A non-exported `addedNotifier` would also have worked, so I was curious if there was a specific motivation behind the choice.

Reply:
Gotcha. My preference is to always have the interface exported if it is referenced by some exported type/function. IMO you should be able to understand how to use the interface just by clicking through the godoc UI, without delving into source.

Reply:
cool, thanks!